model/installation.go

```diff
@@ -71,6 +73,12 @@ type Installation struct {
 	configMergeGroupSequence int64
 }
 
+// PodProbeOverrides contains configuration for overriding pod probe settings.
+type PodProbeOverrides struct {
```

Comment:
Initially attempted to use the provisioner declaration of this type, but bringing that to the model package caused a circular dependency issue. Open to suggestions on centralizing, but 0/5 on this small duplication